test/checkoutput/main.go

```diff
@@ -95,11 +92,11 @@ func (rv *runVerify) validate() error {
 	// TODO: Currently we capture the output of just the stdout. Logs extensively write to
 	// stderr, we need to come up with the right logic for validation
 	if string(output) == rv.verify {
-		fmt.Printf("Validation Success: command : %v \n Matched text: %v\n", rv.run, rv.verify)
+		klog.V(2).Infof("Validation Success: command : %v \n Matched text: %v\n", rv.run, rv.verify)
```

Comment:
We are from the WG Structured Logging - please don't use unstructured logging calls :grin: